packages/docusaurus-theme-redoc/src/theme/Redoc/ServerStyles.tsx

```diff
@@ -39,8 +39,9 @@ export function ServerStyles({
   specProps: SpecProps;
   lightThemeOptions: RedocRawOptions;
   darkThemeOptions: RedocRawOptions;
+  absolute: boolean
 }) {
-  const fullUrl = useBaseUrl(specProps.url, { absolute: true });
+  const fullUrl = specProps.normalizeUrl ? useBaseUrl(specProps.url, { absolute: true }) : specProps.url;
```

Comment:
I don't remember why I added absolute true, but thanks for keeping existing behaviour